packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRow.tsx

```diff
@@ -86,6 +86,18 @@ export function TransactionsTableRow({
       )
     }
 
+    if (isTxExpired(tx)) {
+      return (
+        <div className="flex items-center space-x-1 text-red-400">
+          <XCircleIcon height={14} className="mr-1" />
+          <span>Expired</span>
+          <ExternalLink href={`${getExplorerUrl(sourceChainId)}/tx/${tx.txId}`}>
+            <ArrowTopRightOnSquareIcon height={10} />
+          </ExternalLink>
+        </div>
+      )
+    }
```

Comment:
same comment as above, this can safely be its own component

Reply:
done